PEP 1: Further refine phrasing and organization of PEP-Delegate description (#2257)
python/peps: pep-0001.txt

The final authority for PEP approval is the Steering Council. However, whenever

> a new PEP is put forward, any core developer that believes they are suitably

Reviewer: ```suggestion
a new PEP is put forward, any core developer who believes they are suitably
```

experienced to make the final decision on that PEP may offer to serve as

> the PEP-Delegate for that PEP, and not declined by the Steering Council,

Reviewer 2: ```suggestion
the PEP-Delegate for that PEP, and they should notify the Steering Council of their offer. If not declined by the Steering Council,
```

Author: Since the notification requirements are discussed more specifically two paragraphs below this, I'm not sure we need to discuss that in detail here, where we're just summarizing the basic nature of the PEP-Delegate position rather than how to become one (as discussed below). However, I rewrote it to incorporate the spirit of this, specifying that the steering council "assents to their offer" (which requires that the Steering Council be notified of said offer), and I also broke this up into two sentences as you had it, and fixed the missing word ("if"), as well as some repetition and suboptimal phrasing I noticed in the process. Is that okay?

Reviewer 2: In general, I think that is fine. I would be explicit about notification of the Steering Council is one step toward becoming the PEP-Delegate. Acceptance by the steering council is the second step.

Reviewer 2: Looks good to me.

> they will then have the authority to approve or reject that PEP.

Reviewer 2: ```suggestion
the core developer will then have the authority to approve or reject that PEP.
```

Author: See above. I used "PEP-Delegate" here, as it was more specific and fit better with the rephrasing I ultimately did, but incorporated your idea to split the sentences.

The term "PEP-Delegate" is used under the Steering Council governance model
for the PEP's designated decision maker,

> and recorded in the "PEP-Delegate" field in the PEP's header.

Reviewer: ```suggestion
who is recorded in the "PEP-Delegate" field in the PEP's header.
```
The decision maker is recorded in the field, not the term.

Author: Good catch!

For PEPs written prior to the Steering Council governance model, the field name

> "BDFL-Delegate" will still be kept.

Reviewer: I think this sentence can be removed without loss of meaning.

Author: Agreed; I didn't add it so I didn't want to remove it, but I don't think such administrative details are critical to include in the PEP

Author: ```suggestion
```
